metadata/demographic/population_block_areas_2010_approx.md

```diff
@@ -0,0 +1,70 @@
+# Title
+
+Utah Population Block Areas 2010 Approx
+
+## ID
+
+0b1ca582-e847-4098-8c00-f2b63e00fe02
+
+## Brief Summary
+
+Polygon dataset of Census blocks in populated areas as determined by aerial imagery.
```

Comment:
```suggestion
Polygon dataset of Census blocks trimmed to cover only populated areas in Utah.
```
Making this a little more generic since the methodology is described more below.